jsvg/src/main/java/com/github/weisj/jsvg/nodes/prototype/impl/HasGeometryContextImpl.java

```diff
@@ -23,52 +23,62 @@
 
 import java.awt.geom.AffineTransform;
 import java.awt.geom.Point2D;
+import java.awt.geom.Rectangle2D;
 
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
 import com.github.weisj.jsvg.attributes.value.PercentageDimension;
+import com.github.weisj.jsvg.attributes.TransformBox;
 import com.github.weisj.jsvg.geometry.size.Length;
 import com.github.weisj.jsvg.geometry.size.MeasureContext;
 import com.github.weisj.jsvg.nodes.ClipPath;
 import com.github.weisj.jsvg.nodes.Mask;
 import com.github.weisj.jsvg.nodes.filter.Filter;
 import com.github.weisj.jsvg.nodes.prototype.HasGeometryContext;
 import com.github.weisj.jsvg.parser.AttributeNode;
+import com.github.weisj.jsvg.renderer.ElementBounds;
+import com.github.weisj.jsvg.renderer.RenderContext;
 
 public final class HasGeometryContextImpl implements HasGeometryContext {
 
     private final @Nullable AffineTransform transform;
     private final @NotNull Length transformOriginX;
     private final @NotNull Length transformOriginY;
+    private final @NotNull TransformBox transformBox;
 
     private final @Nullable ClipPath clipPath;
     private final @Nullable Mask mask;
     private final @Nullable Filter filter;
 
+    private final @NotNull Object node;
 
     private HasGeometryContextImpl(@Nullable AffineTransform transform, @NotNull Length transformOriginX,
-            @NotNull Length transformOriginY, @Nullable ClipPath clipPath, @Nullable Mask mask,
-            @Nullable Filter filter) {
+            @NotNull Length transformOriginY, @NotNull TransformBox transformBox, @Nullable ClipPath clipPath,
+            @Nullable Mask mask, @Nullable Filter filter, @NotNull Object node) {
```

Comment:
It should be possible to take an `SVGNode` here instead. I think the fact that `ElementBounds` takes an `Object` parameter is a left over artefact from when it was used for non-related classes, where the nearest superclass was `Object`.